-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-38055/30139

ERROR: Build errors found during clang-tidy run.
```
Geometry/HGCalCommonData/plugins/DDHGCalSiliconRotatedModule.cc:184:35: error: no member named 'layerType' in 'HGCalTypes' [clang-diagnostic-error]
    layerOrient_[k] = HGCalTypes::layerType(layerOrient_[k]);
                                  ^
Suppressed 639 warnings (638 in non-user code, 1 with check filters).
--
Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalSiliconRotatedModule.cc:120:37: error: no member named 'layerType' in 'HGCalTypes' [clang-diagnostic-error]
      layerOrient_[k] = HGCalTypes::layerType(layerOrient_[k]);
                                    ^
Suppressed 2149 warnings (2148 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/Geometry/HGCalCommonData/plugins/DDHGCalSiliconRotatedModule.cc
+++ b/Geometry/HGCalCommonData/plugins/DDHGCalSiliconRotatedModule.cc
@@ -389,6 +389,7 @@ void DDHGCalSiliconRotatedModule::constructLayers(const DDLogicalPart& module, D
   }  // End of loop over blocks
 }
 
+// Position the silicon modules
 void DDHGCalSiliconRotatedModule::positionSensitive(const DDLogicalPart& glog, int layer, DDCompactView& cpv) {
   static const double sqrt3 = std::sqrt(3.0);
   int layercenter = layerOrient_[layer];

